core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.spec.ts

```diff
@@ -103,7 +106,7 @@ class MockPlatformFeatureService {
   }
 }
 
-describe('Conversation skin component', () => {
+fdescribe('Conversation skin component', () => {
```

Comment:
```suggestion
describe('Conversation skin component', () => {
```
Please don't push changes without running linters.

Reply:
Fixed.